multi-agent/lib/mas/core/field_hints.py

```
@@ -71,18 +77,18 @@ class ActionHint(BaseModel):
         description="Whether the action supports search filtering (has search_regex param)"
     )
-    on_success: Optional["ActionHint"] = Field(
+    on_success: "ActionHint | None" = Field(
         default=None,
         description="Action to fire after this action succeeds. "
                     "The chained action receives its own dependencies from the form. "
                     "Skipped when any required dependency value is empty.",
     )
 
-    def model_dump(self, **kwargs) -> Dict[str, Any]:
+    def model_dump(self, **kwargs) -> dict[str, Any]:
```

Ruff (0.15.21)

[warning] Missing type annotation for `**kwargs`

(ANN003)